src/storage/exec/UpdateNode.h

```diff
@@ -262,7 +274,18 @@ class UpdateTagNode : public RelNode<VertexID> {
                     }
                     auto ni = indexKey(partId, vId, nReader.get(), index);
                     if (!ni.empty()) {
-                        batchHolder->put(std::move(ni), "");
+                        if (planContext_->env_->checkRebuilding(planContext_->spaceId_,
+                                                                partId, indexId)) {
+                            auto modifyKey = OperationKeyUtils::modifyOperationKey(partId,
+                                                                                   std::move(ni));
+                            batchHolder->put(std::move(modifyKey), "");
+                        } else if (planContext_->env_->checkIndexLocked(planContext_->spaceId_,
+                                                                        partId, indexId)) {
+                            LOG(ERROR) << "The part have locked";
```

Comment:
Why we don't update the `onFlyingRequest_` in `update`?

Reply:
missed